addon/mixins/controller-history.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+	
+	actions: {
+		backButton(context) {
```

Comment:
I don't really like the naming of this action, as it doesn't clearly illustrates what this action does.